modules/virt-creating-linux-bridge-nncp.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
```

Comment:
See my other comment about including this content in this assembly. If you do have to include it, it probably makes more sense to reuse the [existing module](https://docs.openshift.com/container-platform/4.10/networking/k8s_nmstate/k8s-nmstate-updating-node-network-config.html#virt-example-bridge-nncp_k8s_nmstate-updating-node-network-config) rather than creating a new one.